components/gallery/GalleryItem.vue

```diff
@@ -12,12 +12,32 @@
       <div class="column is-two-fifths">
         <div class="is-relative">
           <a
-            v-if="canPreview"
+            v-if="canPreview && !isResources"
             class="fullscreen-button is-justify-content-center is-align-items-center"
             @click="isFullscreen = true">
             <NeoIcon icon="expand" />
           </a>
+          <div v-if="isResources" class="gallery-item-carousel">
+            <o-carousel
```

Comment:
can't we reuse `CarouselMedia`?

Reply:
we need to adjust `CarouselMedia` first to disable `nuxt-link` tag